_content/monographs/kamma-in-buddhism_buddhadasa.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Kamma in Buddhism"
+authors:
+  - buddhadasa
+translator: Santikaro
+external_url: "https://www.suanmokkh.org/system/books/files/000/000/089/original/Buddhadas-Kamma-in-buddhism.pdf?1599464247"
+drive_links:
+  - "https://drive.google.com/file/d/1OdOqbmOuzO48q0a7tR508jvGMIgo2REX/view?usp=drive_link"
+formats: [pdf]
+course: view
+tags:
+  - karma
+  - rebirth
+year: 2017
+publisher: Liberation Park
+address: "Wisconsin"
+volume: 
+series: Messages from Suan Mokkh Series
+number: 4
+pages: 22
+---
+
+> We should see the
+truth that the mind that performs a deed is kamma itself and the
+subsequent mind is the result (vipāka) of that kamma. Other
+results that follow it are only uncertain by-products, since they
+may or may not occur, or do not keep up with our expectations
+due to other interfering factors.
+
+In this monograph, the author explains kamma (karma) as it is understood in Buddhism and how to use kamma (action) in a way that leads to the end of action and results in nibanna.
```

Comment:
Idk man... it's so short I'm inclined to move this to essays... what do you think?